root/user/profile/layout.tt

```diff
@@ -17,19 +17,20 @@
 [%- info_links.push(
     ['tags', simple_link(c.uri_for_action("/user/tags", [ user.name ]), l('Tags'))],
     ['ratings', simple_link(c.uri_for_action("/user/ratings", [ user.name ]), l('Ratings'))],
-    ['open_edits', simple_link(c.uri_for_action("/user/edits/open", [ user.name ]), l('Open Edits'))],
-    ['all_edits', simple_link(c.uri_for_action("/user/edits/all", [ user.name ]), l('All Edits'))],
```

Comment:
These are linked from the main profile page, and aren't actually profile tabs but links to the normal edit listings, suggesting they don't really belong here.